src/Screen/DetailsScreen/index.tsx

```diff
@@ -1,42 +1,51 @@
-import React, {FC, useEffect, useState} from 'react';
-import {View, Text, Linking, ActivityIndicator, FlatList, TouchableOpacity, ListRenderItem} from 'react-native';
+import React, { FC, useEffect, useState } from 'react';
+import {
+  View,
+  Text,
+  Linking,
+  ActivityIndicator,
+  FlatList,
+  TouchableOpacity,
+  ListRenderItem,
+} from 'react-native';
 import { createStyles } from './style';
-import {DetailsScreenProps, itemLinks} from './types';
+import { DetailsScreenProps, ItemLinks } from './types';
 import { useThemeAwareObject } from '@theme/ThemeAwareObject.hook';
 import { useTranslation } from 'react-i18next';
 import { useAppDispatch, useAppSelector } from '@redux/hooks';
 import { requestAirportDetails } from '@redux/action/airportDetails';
 import { getDetailsAirport } from '@redux/selectors';
 import { createArrayLinks } from '@screen/DetailsScreen/helper';
+import { DetailsScreenView } from '@screen/DetailsScreen/DetailsScreenView';
+import { InitialDetailsAirportStateTypes } from '@redux/reducer/types';
 
 export const DetailsScreen: FC<DetailsScreenProps> = (props) => {
-
   const Styles = useThemeAwareObject(createStyles);
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
-  const [linksAirports, setLinksAirports] = useState<itemLinks[]>([]);
   const { pending, airportData } = useAppSelector(getDetailsAirport);
 
   useEffect(() => {
     dispatch(requestAirportDetails(props.route.params.iata));
   }, []);
 
-  useEffect(() => {
+  const renderItem: ListRenderItem<ItemLinks> = ({ item }) => {
+    return (
+      <TouchableOpacity onPress={() => Linking.openURL(item.link)}>
+        <Text style={Styles.link}>{item.name}</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  const linksAboutAirport = () => {
+    const list: ItemLinks[] = [];
```

Comment:
I am not sure that we have to add type here. it is an empty array and next you are pushing the correct object with a defined type. I think as result we could see in list ItemLinks[], but let's check it

Reply:
I tried to delete this type declaration, but ts compilator said me about the problem "TS7034: Variable 'list' implicitly has type 'any[]' in some locations where its type cannot be determined"